pandas/tests/arrays/test_array.py

```diff
@@ -70,41 +65,42 @@
         (
             pd.DatetimeIndex(["2000", "2001"]),
             np.dtype("datetime64[ns]"),
-            DatetimeArray._from_sequence(["2000", "2001"]),
+            DatetimeArray._from_sequence([pd.Timestamp("2000"), pd.Timestamp("2001")]),
```

Comment:
Can you revert the changes to the tests? 
(or otherwise ensure we keep coverage of converting from non-Timestamp scalars)

Reply:
i was on the fence about this, glad you tipped the scales.  will update.

Reply:
i was on the fence about this, glad you tipped the scales.  will update.